h/resources.py

```diff
@@ -44,6 +45,7 @@ class Root(Resource):
         (Allow, 'group:__admin__', 'admin_badge'),
         (Allow, 'group:__admin__', 'admin_groups'),
         (Allow, 'group:__staff__', 'admin_groups'),
+        (Allow, security.Authenticated, 'authenticated'),
```

Comment:
This is a bit of an abuse of the ACL system. Just raise a 404 (or whatever) from within the view if `request.authenticated_user` is `None`.